Source/WebCore/xml/XMLErrors.cpp

```diff
@@ -63,13 +63,13 @@ void XMLErrors::handleError(ErrorType type, const char* message, int lineNumber,
 
 void XMLErrors::handleError(ErrorType type, const char* message, TextPosition position)
 {
-    if (type == fatal || (m_errorCount < maxErrors && (!m_lastErrorPosition || (m_lastErrorPosition->m_line != position.m_line && m_lastErrorPosition->m_column != position.m_column)))) {
+    if (type == ErrorType::Fatal || (m_errorCount < maxErrors && (!m_lastErrorPosition || (m_lastErrorPosition->m_line != position.m_line && m_lastErrorPosition->m_column != position.m_column)))) {
         switch (type) {
-        case warning:
-            appendErrorMessage("warning"_s, position, message);
+        case ErrorType::Warning:
+            appendErrorMessage("ErrorType::Warning"_s, position, message);
```

Comment:
This patch probably shouldn’t change the error message. We should just keep the “warning” string, especially since you’re keeping "error" as the message below on line 73.